diff --git a/app/controllers/dmsf_upload_controller.rb b/app/controllers/dmsf_upload_controller.rb
index b2590f2..2515b47 100644
--- a/app/controllers/dmsf_upload_controller.rb
+++ b/app/controllers/dmsf_upload_controller.rb
@@ -132,6 +132,7 @@ class DmsfUploadController < ApplicationController
 
   def commit_files_internal(commited_files)
     @files, failed_uploads = DmsfUploadHelper.commit_files_internal(commited_files, @project, @folder, self)
+    call_hook(:dmsf_upload_controller_after_commit, { files: @files })
     respond_to do |format|
       format.js
       format.api  { render_validation_errors(failed_uploads) unless failed_uploads.empty? }
diff --git a/app/views/dmsf/_main.html.erb b/app/views/dmsf/_main.html.erb
index 923748c..2f32e0c 100644
--- a/app/views/dmsf/_main.html.erb
+++ b/app/views/dmsf/_main.html.erb
@@ -130,3 +130,7 @@
     });
   <% end %>
 <% end %>
+
+<% late_javascript_tag do %>
+  "$('#ajax-indicator').hide();"
+<% end %>
diff --git a/app/views/dmsf_upload/upload_files.html.erb b/app/views/dmsf_upload/upload_files.html.erb
index 7af5684..d0a7dfc 100644
--- a/app/views/dmsf_upload/upload_files.html.erb
+++ b/app/views/dmsf_upload/upload_files.html.erb
@@ -45,7 +45,8 @@
       <% end %>
     <% end %>
     <div class="form-actions">
-      <%= submit_tag l(:label_upload), class: 'button-positive' %>
+      <%= submit_tag l(:label_upload), class: 'button-positive',
+                     onclick: "$('#ajax-indicator').show();" %>
     </div>
   <% end %>
 <% end %>
