Index: db/migrate/062_insert_builtin_roles.rb
===================================================================
--- db/migrate/062_insert_builtin_roles.rb	(revision 12244)
+++ db/migrate/062_insert_builtin_roles.rb	(working copy)
@@ -1,16 +0,0 @@
-class InsertBuiltinRoles < ActiveRecord::Migration
-  def self.up
-    Role.reset_column_information
-    nonmember = Role.new(:name => 'Non member', :position => 0)
-    nonmember.builtin = Role::BUILTIN_NON_MEMBER
-    nonmember.save
-
-    anonymous = Role.new(:name => 'Anonymous', :position => 0)
-    anonymous.builtin = Role::BUILTIN_ANONYMOUS
-    anonymous.save
-  end
-
-  def self.down
-    Role.destroy_all 'builtin <> 0'
-  end
-end
