Index: db/migrate/062_insert_builtin_roles.rb
===================================================================
--- db/migrate/062_insert_builtin_roles.rb	(revision 12244)
+++ db/migrate/062_insert_builtin_roles.rb	(working copy)
@@ -2,11 +2,11 @@
   def self.up
     Role.reset_column_information
     nonmember = Role.new(:name => 'Non member', :position => 0)
-    nonmember.builtin = Role::BUILTIN_NON_MEMBER
+    #nonmember.builtin = Role::BUILTIN_NON_MEMBER
     nonmember.save
 
     anonymous = Role.new(:name => 'Anonymous', :position => 0)
-    anonymous.builtin = Role::BUILTIN_ANONYMOUS
+    #anonymous.builtin = Role::BUILTIN_ANONYMOUS
     anonymous.save
   end
 
